src/polyfem/state/StateSolveNonlinear.cpp

```diff
@@ -75,7 +75,7 @@ namespace polyfem
 
 	void SolveData::update_dt()
 	{
-		if (inertia_form)
+		if (time_integrator) // if is time dependent
```

Comment:
are you sure? ignore inertia is incremental load so it shouldn't do the dt thing

Reply:
I see your point if ignore_inertia really means do static with an incremental load. I was thinking it meant do time-dependent without the inertia term.

Reply:
Reverted it back to if(inertia_form)